src/typings.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import type { Element } from 'react';
+
+export type Styles = {
+  [string]: string,
+};
+
+type ReactStyles = {
+  [string]: string | number,
+};
```

Comment:
It can be a bit confusing, because when you supply `height: 20`, we don't append `px` (but people may expect us to do it, because React does), and height doesn't get set to desired value